android/app/build.gradle

```diff
@@ -49,7 +49,7 @@ dependencies {
     implementation 'org.mapsforge:vtm-android:0.9.2:natives-x86'
     implementation 'org.mapsforge:vtm-android:0.9.2:natives-x86_64'
     implementation('org.mapsforge:vtm-jts:0.9.2') {
-      exclude group: 'com.vividsolutions', module: 'jts'
+      exclude group: 'org.locationtech', module: 'jts'
```

Comment:
This isn't needed since VTM uses JTS (LocationTech) from version [0.10.0](https://github.com/mapsforge/vtm/blob/master/docs/Changelog.md).

Reply:
Even better, thx.